openedx/core/djangoapps/user_api/preferences/tests/test_views.py

```diff
@@ -532,7 +531,7 @@ def test_create_preference_other_user(self, api_client, user):
         self._set_url("new_key")
         client = self.login_client(api_client, user)
         new_value = "new value"
-        self.send_put(client, new_value, expected_status=403 if user == "staff_user" else 404)
```

Comment:
Again, I'm not clear on why we want this for gets, but not other actions?  Same for two below.

Reply:
This is expected.  See comment above.  This is not for a non-existent user.